src/System.Runtime.Serialization.Xml/tests/System.Runtime.Serialization.Xml.Tests.csproj

```diff
@@ -19,6 +19,18 @@
     <Compile Include="MyResolver.cs" />
     <Compile Include="XmlDictionaryWriterTest.cs" />
   </ItemGroup>
+  <!-- Performance tests require v5.0 portable tools -->  
```

Comment:
This section is no longer needed. Some changes to buildtools were made that allowed us to leave out this entire section:
```
  <!-- Performance tests require v5.0 portable tools -->  
  <PropertyGroup Condition="'$(RunPerfTestsForProject)' == 'true'">  
    <TargetFrameworkVersion>v5.0</TargetFrameworkVersion>  
    <TargetFrameworkProfile />  
  </PropertyGroup>  
```
I'll update the docs accordingly.